db/migrate/20220826124130_create_states.rb

```diff
@@ -0,0 +1,10 @@
+class CreateStates < ActiveRecord::Migration[6.1]
+  def change
+    create_table :states do |t|
```

Comment:
i forgot to add in the past review, Punch is Multitenant, so we should scope this to the company